invokeai/frontend/.gitignore

```diff
@@ -25,4 +25,7 @@ dist-ssr
 *.sw?
 
 # build stats
-stats.html
\ No newline at end of file
+stats.html
+
+# Yarn
+.yarn
```

Comment:
And this is a yarn v2+ file also?

Reply:
Yes. Refer to the other comment. This'll still be needed in the commit, but has been fine-tuned to Yarn's recommendations.